ikvmocr

```diff
@@ -562,8 +562,10 @@ def main(screenshot_filenames, glyph_db_filename):
 
 
 if __name__ == '__main__':
+    ikvmcr_data_dir = f'{os.environ.get("XDG_DATA_HOME",
+        "~/.local/share")}/ikvmocr'
```

Comment:
I think it's nicer to put `LIBDIR = os.environ.get("XDG_DATA_HOME", "~/.local/share")}/ikvmocr'` at the top of the file. Anyone importing this (not likely) can then still update that if needed. And the rest of the code doesn't need to concern itself with passing the global lib path around.

Reply:
I believe you commented on one of my other PRs that you didn't want me to indent the second argument to be under the first, but instead wanted me to just use a single indent on the the new line. But I could be mistaken.

Reply:
?

Yes. Which the code above doesn't comply with. So you're not mistaken. Not sure what you're seeing differently from me here.

I see:

(1) Violation of said rule, which would have been avoided by doing this (or even better, the next one):
```
    ikvmcr_data_dir = f'{os.environ.get(
        "XDG_DATA_HOME", "~/.local/share")}/ikvmocr'
```
(2) Syntax error (with python 3.10), which would have been avoided by doing this:
```
    ikvmcr_data_dir = (
        f'{os.environ.get("XDG_DATA_HOME", "~/.local/share")}/ikvmocr')
```

Reply:
Thanks for explaining, this was fixed since this doesn't have to be indented anymore at the top of the file. But I'll keep this in mind for future PRs.